agent/lib_guzzle6.h

```diff
@@ -12,14 +12,17 @@
 #include "php_includes.h"
 
 /*
- * Purpose : Performs tasks that we need performed on MINIT in the Guzzle 6
+ * Purpose : Performs tasks that we need performed on MINIT in the Guzzle 6 and 7
  *           instrumentation.
```

Comment:
Needs to be clang formatted

```suggestion
 * Purpose : Performs tasks that we need performed on MINIT in the Guzzle 6 and
 * 7 instrumentation.
```